CalculatorG.rb

```diff
@@ -0,0 +1,237 @@
+# Project Calculator (gold)
+
+puts "Choose from the list what operation you would like to do:\n "
+puts " add (+)\n sub (-)\n mult (*)\n div(/)\n exponent (^, **)\n mod (%)"
+user_input = gets.chomp
+
+# Function for addition……………………………………………………………………………………………………………………………………………………
+def add
+    puts "Ok, lets do addition. Type in first number"
+    first_number = gets.chomp
+        while first_number != "0" && first_number.to_f == 0
```

Comment:
Adding a comment here would be good to understand exactly what case these conditionals are checking for.